First_project/First_project/Profile/ProfileViewController.swift

```diff
@@ -29,25 +31,32 @@ class ProfileViewController: UIViewController {
         view.backgroundColor = .systemGroupedBackground
         navigationController?.navigationBar.prefersLargeTitles = true
         
-        view.addSubview(areaView)
-        
-        areaView.topAnchor.constraint(equalTo: view.safeAreaLayoutGuide.topAnchor).isActive = true
-        areaView.bottomAnchor.constraint(equalTo: view.safeAreaLayoutGuide.bottomAnchor).isActive = true
-        areaView.leftAnchor.constraint(equalTo: view.safeAreaLayoutGuide.leftAnchor).isActive = true
-        areaView
-            .rightAnchor.constraint(equalTo: view.safeAreaLayoutGuide.rightAnchor).isActive = true
+        [areaView].forEach {
+            $0.translatesAutoresizingMaskIntoConstraints = false
+            view.addSubview($0)
+        }
         
-        //        let widht:CGFloat = 60
-        //
-        //        areaView.widthAnchor.constraint(equalToConstant: widht).isActive = true
-        //        areaView.heightAnchor.constraint(equalToConstant: widht).isActive = true
+        NSLayoutConstraint.activate([
+            areaView.topAnchor.constraint(equalTo: view.safeAreaLayoutGuide.topAnchor),
+            areaView.bottomAnchor.constraint(equalTo: view.safeAreaLayoutGuide.bottomAnchor),
+            areaView.leftAnchor.constraint(equalTo: view.safeAreaLayoutGuide.leftAnchor),
+            areaView.rightAnchor.constraint(equalTo: view.safeAreaLayoutGuide.rightAnchor)
+        ])
+
         let shareProfileButton = UIBarButtonItem(image: UIImage(systemName: "text.justify"),  style: .plain, target: self, action: #selector(settingsProfile))
         shareProfileButton.tintColor = .black
         
         let settingsProfileButton = UIBarButtonItem(image: UIImage(systemName: "square.and.arrow.up"),  style: .plain, target: self, action: #selector(shareProfile))
         settingsProfileButton.tintColor = .black
         
         navigationItem.rightBarButtonItems = [shareProfileButton, settingsProfileButton]
+        
+        view.addSubview(profileHeader)
+    }
+    
+    override func viewDidLayoutSubviews() {
+        super.viewDidLayoutSubviews()
+        profileHeader.frame = CGRect(x: 0, y: 143, width: view.bounds.width, height:  300)
```

Comment:
Why do you use frames instead of constraints? Better to stick with one approach.